controllers/healthcheck_test.go

```diff
@@ -0,0 +1,31 @@
+package controllers
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/gorilla/mux"
+	"github.com/prest/prest/testutils"
+)
+
+func TestHealthStatus(t *testing.T) {
+	var testCases = []struct {
+		description string
+		url         string
+		method      string
+		status      int
+	}{
+		{"Get databases without custom where clause", "/_health", "GET", http.StatusOK},
+	}
+
+	router := mux.NewRouter()
+	router.HandleFunc("/_health", HealthStatus).Methods("GET")
+	server := httptest.NewServer(router)
+	defer server.Close()
+
+	for _, tc := range testCases {
+		t.Log(tc.description)
+		testutils.DoRequest(t, server.URL+tc.url, nil, tc.method, tc.status, "HealthStatus")
+	}
```

Comment:
Not sure if this is a pattern from pREST but any reason to not use the native `.t.Run(…)`?

```suggestion
	for _, tc := range testCases {
		t.Run(tc.description, function (t *testing.T) {
				testutils.DoRequest(t, server.URL+tc.url, nil, tc.method, tc.status, "HealthStatus")
		}
	}
```